### PR TITLE
Remove DO and contractor fields from summary page

### PR DESCRIPTION
**Pull request by chiaberry**
## Associated issues

First half of
https://github.com/cityofaustin/atd-data-tech/issues/9770 

Now that we have a contractor table, and will eventually be dropping these columns from the moped_project table, we should remove these fields from the summary page so users do not use deprecated fields. 

## Testing
**URL to test:** 

https://deploy-preview-747--atd-moped-main.netlify.app/moped/projects/1865

**Steps to test:**

The summary page should not give users an option to input contractor names or DO#. 

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js [hunk 1]
```diff
@@ -260,10 +238,11 @@ const ProjectSummary = ({ loading, error, data, refetch }) => {
             </Grid>
             <Grid container spacing={0}>
               <Grid item xs={6}>
-                <ProjectSummaryWorkOrders
-                  classes={classes}
-                  project={data?.moped_project?.[0]}
+                <ProjectSummaryInterimID
```
Comment (chiaberry) [inline moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummary.js:241]: I switched these around because the empty space was killing me.

**Review by mateoclarke — CHANGES_REQUESTED** [review]
This seems unrelated to your PR (maybe another recent merge to staging?) but I'm seeing this error in the Deploy Preview instance...

Screen Shot 2022-08-11 at 12 09 57 PM

**Comment by chiaberry** [comment]
You are right! I just merged the other one to staging so I pulled main into this one. hopefully I've fixed it now

**Review by mateoclarke — APPROVED** [review]
🚢 💨 ✨

**Review by patrickm02L — APPROVED** [review]
Looks good. 🤘

Screen Shot 2022-08-11 at 3 40 30 PM

**Review by johnclary — APPROVED** [review]
🚢 🚢 🚢 🚢 🚢 🚢

**Pull request merged**